main.ino

```diff
@@ -0,0 +1,26 @@
+#include "bno.h"
+#include "motor.h"
+#include "movement.h"
+#include "pixy.h"
+#include <Wire.h>
+#include <Adafruit_Sensor.h>
+#include <Adafruit_BNO055.h>
+
+void setup(){
+
+}
+
+void loop(){
+    Bno bno_sensor1;
```

Comment:
I don seem very optimal to declare an object of type BNo and Movements each iteration of the loop.